components/ide/code/leeway.Dockerfile

```diff
@@ -40,12 +40,15 @@ RUN curl -fsSL https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh |
 ENV PATH $NVM_DIR/versions/node/v$NODE_VERSION/bin:$PATH
 
 ENV GP_CODE_COMMIT 37f3ceefb1c5ca4e492334316c6b65cf7e665a2d
-RUN mkdir gp-code \
-    && cd gp-code \
-    && git init \
-    && git remote add origin https://github.com/gitpod-io/vscode \
-    && git fetch origin $GP_CODE_COMMIT \
-    && git reset --hard FETCH_HEAD
+#RUN mkdir gp-code \
+#    && cd gp-code \
+#    && git init \
+#    && git remote add origin https://github.com/gitpod-io/vscode \
+#    && git fetch origin $GP_CODE_COMMIT \
+#    && git reset --hard FETCH_HEAD
```

Comment:
<details>
<summary>:warning: <strong>Commented-out Code is Committed.</strong> (click for details)</summary>

This pull request contains commented code which is not recommended when a version control repository (e.g., GitHub) is used. It is recommended to delete all commented instructions before committing to VCS.

**[More details](https://confluence.devfactory.com/x/yAV7HQ)**

## The improvement suggestion in this case
Consider avoiding committing commented code.
</details>

[//]: # (<a id="f1caed9f-4626-459c-acf6-9411e0935d22" data-finding-id="494601672" href="none" codereview-uuid-reserved-tag data-business-id="d2a596db-d8bf-43dd-b073-6d1bacb7fbe5" />)